README.md

```diff
@@ -2,11 +2,14 @@
 
 [![godoc reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/libdns/hetzner)
 
-This package implements the libdns interfaces for the [Hetzner DNS API](https://dns.hetzner.com/api-docs).
+This package implements the [libdns](https://github.com/libdns/libdns) interfaces for the Hetzner DNS APIs.
```

Comment:
Add link to new docs: https://docs.hetzner.cloud/reference/cloud#dns

Reply:
The new docs are linked to in the second bullet point, does that suffice?